packages/amplify_authenticator/lib/src/utils/country_code.dart

```diff
@@ -25,6 +25,7 @@ class Country {
 
 /// List of country information
 const countryCodes = [
+  Country('US', '1', CountryResolverKey.us$),
```

Comment:
Would be fine with a separate PR for this, but maybe we order these by the code if we want US/+1 as the first on the list?

Reply:
This could come as a fast follow, but I was hoping we could use the device locale to attempt to set the default?